src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/UntypedObject.cs

```diff
@@ -45,4 +51,94 @@ public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
             yield return new[] { TexlStrings.IndexArg1, TexlStrings.IndexArg2 };
         }
     }
+
+    internal abstract class UntypedOrJSONConversionFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool SupportsParamCoercion => false;
+
+        public override bool HasTypeArgs => true;
+
+        public override bool ArgIsType(int argIndex)
+        {
+            return argIndex == 1;
+        }
+
+        private static readonly ISet<DType> UnsupportedJSONTypes = new HashSet<DType> { DType.Color, DType.Void, DType.Time, DType.ObjNull };
```

Comment:
>DType.Time [](http://example.com/codeflow?start=113&length=10)

Why is Time not supported? Something like `TimeValue(ParseJSON("""13:35:57"""))` works fine today